cmip6_downscaling/methods/gard/tasks.py

```diff
@@ -0,0 +1,281 @@
+from dataclasses import asdict
+
+import dask
+import numpy as np
+import pandas as pd
+import xarray as xr
+import xesmf as xe
+import zarr
+from carbonplan_data.metadata import get_cf_global_attrs
+from prefect import task
+from skdownscale.pointwise_models import (  # AnalogRegression,; PureAnalog,; PureRegression,
+    PointWiseDownscaler,
+)
+from skdownscale.pointwise_models.utils import default_none_kwargs
+from upath import UPath
+
+from cmip6_downscaling import __version__ as version, config
+
+from ..common.bias_correction import bias_correct_gcm_by_method
+from ..common.containers import RunParameters, str_to_hash
+from ..common.utils import zmetadata_exists
+from .utils import add_random_effects, get_gard_model
+
+scratch_dir = UPath(config.get("storage.scratch.uri"))
+intermediate_dir = UPath(config.get("storage.intermediate.uri")) / version
+results_dir = UPath(config.get("storage.results.uri")) / version
+use_cache = config.get('run_options.use_cache')
+
+good_fit_predict_chunks = {'lat': 24, 'lon': 24, 'time': 10957}
+
+
+@task(tags=['dask-resource:taskslots=1'], log_stdout=True)
```

Comment:
@norlandrhagen - can you confirm that this is still the correct set of task tags?